src/lib/Server.js

```diff
@@ -59,6 +60,11 @@ const isPasswordValid = (password) => {
   return false;
 };
 
+const cronJobEveryMinute = async () => {
```

Comment:
Why is this not using [setInterval()](https://developer.mozilla.org/en-US/docs/Web/API/setInterval)?

Why is the interval hardcoded to run every single minute of every day if the expiry time is hardcoded to 23:59.50 anyway?

Reply:
It doesn't matter whether to use setInterval or setTimeout. There is no difference between them in this implementation. If it is so necessary, we can change it to setInterval


If you run the function once a day, there will be a lag between the blocking time and the actual time when the function was run. And it can be another 24 hours if docker was started at 23:59:49.
And now there is a lag of only 1 minute. 

You can probably make it 10 minutes. Then the lag will be 10 minutes from the time when you need to lock to the time of actual locking.